packages/assets-controllers/src/NftController.test.ts

```diff
@@ -114,14 +123,17 @@ function setupController({
   return {
     assetsContract,
     nftController,
-    network,
     onNftAddedSpy,
     preferences,
     messenger,
```

Comment:
It looks like this messenger was only used for the network controller. Could this be removed now?